src/ui/page/error/renderingError/renderingError.tsx

```diff
@@ -0,0 +1,54 @@
+import type { FC } from 'react'
+import { useCallback } from 'react'
+import { useNavigate } from 'react-router-dom'
+import classNames from 'classnames'
+import { useTranslation } from 'react-i18next'
+import { useAppStore } from '@/ui/store/appStore'
+import '@/ui/page/error/i18n/index'
+import './renderingError.scss'
+import { Button } from '@/ui/component/button/button'
+
+export const RenderingError: FC = () => {
+  const { t } = useTranslation('error')
+  const theme = useAppStore(store => store.theme)
+  const navigate = useNavigate()
+
+  const handleBackToHome = useCallback((): void => navigate('/'), [navigate])
+  const handleContactClick = useCallback(() => {
```

Comment:
`handleBackToHome` and `handleContactClick` should be harmonized with or without `click`